docs/developer/order/overview.mdx

```diff
@@ -28,3 +28,14 @@ Several channel settings affect how orders behave in Saleor:
 - **[`markAsPaidStrategy`](/api-reference/miscellaneous/objects/order-settings.mdx#ordersettingsmarkaspaidstrategymarkaspaidstrategyenum---)**: Determines whether a Payment or Transaction is created when an order is manually marked as paid in Saleor Dashboard or using [`orderMarkAsPaid`](/api-reference/orders/mutations/order-mark-as-paid) mutation.
 - **[`includeDraftOrderInVoucherUsage`](/api-reference/miscellaneous/objects/order-settings.mdx#ordersettingsincludedraftorderinvoucherusageboolean---)**: Determines if vouchers applied on draft orders should count toward voucher usage limits.
 
+Order settings are stored in the channel object and in the store object. On the dashboard, you can find them in the channel settings or in the order list under the settings icon in the upper right corner
```

Comment:
Question: What do you mean by `Order settings are stored in the channel object and in the store object.`. You mean in the DB? I think we shouldn't mention that, it's not relevant to Saleor users

Reply:
@IKarbowiak  I mean in the API. Settings related to order behavior 
on channel like: 
stockSettings
orderSettings
paymentSettings

and on shop: 
fulfillmentAutoApprove
fulfillmentAllowUnpaid

and all of this affects how orders will work.

Reply:
Hm. So I think it's not clear, according to the above `Settings` section, all mentioned settings are defined on channel level.
IMO if we want to mention that something is on store level we should mention what settings we are referring to.